README.md

```diff
@@ -1,45 +1,49 @@
 
 ## How to install
-### Manual (Necessary for Mac and Linux - Works on Windows too)
+### Manual (Necessary for Mac and Linux - works on Windows too)
 **Before following any installation steps**, make sure you [install Node.js](https://nodejs.org/en/).
-To install LiveBot, just download as a zip file, which you can do by pressing the green button above that has `"Code"` written inside. Once you have downloaded and extracted the zip file, go into the directory with a command prompt or shell window. In the shell window, enter the command `npm install`. Make sure that the terminal window is running as administrator. Once this process is finished - it may take some time - type `npm start`. 
+You can install LiveBot in many ways such as Git, but the easiest way is to just download as a compressed (zip) file, which you can do by pressing the green button above that has `Code` written on it. Once you have downloaded and extracted the zip file, go into the directory with a command prompt or shell window. In the shell window, enter the command `npm install` to install the needed dependencies. Make sure that the terminal window is running as Administrator (or root, for Linux people). <!-- is this necessary? --> Once this process is finished — it may take some time — type `npm start`. 
 
-### Easy (Windows Only)
-If you would like to just download and open an exe file, find the releases on the right side of this page, and download the latest release. You will need to download the source code and the exe file, then place the executable in the source code folder. 
+### Easy (Windows only)
+If you would like to just download and open an executable file, find the releases on the right side of this page, and download the latest release. You will need to download the source code and the executable file, then place the executable in the source code folder. 
 
 ## Some features:
 ### Token switcher (User menu)
-This box is how you log into a profile after already logging in. To log into a token by default, put it into this box then press the save button. Otherwise if you are just logging into a token just for a look, input the token into the box then press `enter`.<br>
+This box is how you log into a profile after already logging in. To log into a token by default, put it into this box then press the save button. Otherwise if you are just logging into a token just for a look, input the token into the box then press `enter`.
+
 The token changer used to be at the top of the main screen, but has moved to the user options menu.
-<img src='https://i.imgur.com/1rnEpQD.png' style='border-radius: 10px;'>
-</br>
 
-### Barry!
+![token changer](https://i.imgur.com/1rnEpQD.png)
+

```

Comment:
Again, would like to keep my border radius. Also this says Token changer

Reply:
it's actually the token changer image, just Git displayed it below Barry in the diff

Reply:
Ah